newsroom/utils.py

```diff
@@ -196,9 +196,9 @@ def date_short(datetime):
 def get_agenda_dates(agenda: Dict[str, Any], date_paranthesis: bool = False) -> str:
     start = parse_date_str(agenda.get("dates", {}).get("start"))
     end = parse_date_str(agenda.get("dates", {}).get("end"))
+    all_day = agenda.get("dates", {}).get("all_day", False)
 
-    if start.time() == datetime.min.time() or end.time() == datetime.min.time():
-        # if start time or end time is not present
+    if all_day:
```

Comment:
all day can be set also for multi day events

Reply:
so in this case, we have to show both dates `start and end` right ?

Reply:
exactly